newfile.bib

```diff
@@ -0,0 +1,11 @@
+
+@article{ Kline19
+author = {Kline, Patrick and Walters, Christopher R.},
+title = {On Heckits, LATE, and Numerical Equivalence},
```

Comment:
Just for the "title" field in bib entries: You have to protect capitalized words.  So here, title={On {H}eckits, {LATE}, and numerical equivalence}